src/SelfInitializingFakes/SelfInitializingFakes.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net40;netstandard1.6</TargetFrameworks>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyName>SelfInitializingFakes</AssemblyName>
+    <PackageId>SelfInitializingFakes</PackageId>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>

```

Comment:
The `GenerateAssembly*Attribute` elements are only necessary if you keep the assembly attributes in the code, but if you remove AssemblyInfo.cs you can get rid of them and instead specify the version with `<Version>x.y.z</Version>`. It controls all assembly version attributes, and also the package version. I think `/p:Version=x.y.z` has no effect on the assembly version if you disabled the attribute generation.